examples/Example/src/prot_reanimated/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { StyleSheet, View, Text } from 'react-native';
+import { GestureMonitor, Gesture } from 'react-native-gesture-handler';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withSpring,
+} from 'react-native-reanimated';
+import { useAnimatedGesture } from '../useAnimatedGesture';
+import { useRef } from 'react';
+
+function Draggable() {
+  const [a, sa] = useState(1);
```

Comment:
This should be renamed.

Reply:
Fixed in 6d68c043871fe9eba7ef33f66552ef7d133e044a